_j/cuda_unknown_tf.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Unhandled TensorFlow release X, gracefully giving up recommending based TensorFlow based on CUDA version Y
```

Comment:
what does the second part of this sentence mean?

Reply:
Thanks, adjusted in 82627dc6.